src/schema/v2/me/myBids.ts

```diff
@@ -122,15 +122,15 @@ export const MyBids: GraphQLFieldConfig<void, ResolverContext> = {
     ])
 
     // Map over response to gather all sale IDs
-    const causalityLots = causalityResponse.lotStandingConnection.edges.map(
-      ({ node }) => node
-    )
+    const causalityLots = (
+      causalityResponse?.lotStandingConnection?.edges ?? []
+    ).map(({ node }) => node)
     const causalitySaleIds = causalityLots.map((node) => node.lot.saleId)
     const registeredSaleIds = registeredSalesResponse.body.map(
       (sale) => sale._id
     )
     const watchedSaleIds = watchedSaleArtworksResponse.body.map(
-      (artwork) => artwork.sale_id
+      (artwork) => artwork.artwork.sale_ids[0]
```

Comment:
`artwork.artwork` - the English language is insufficient haha

Reply:
Should be `saleArtwork.artwork` -- will fix in my current PR!